From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Thu, 31 Aug 2017 07:11:39 -0400
Subject: Skip test that started failing with 3.25

That way we can still benefit by running all the other tests.

The test still fails on mips but passes on other arches
as of September 2018.
https://buildd.debian.org/status/logs.php?pkg=mutter&ver=3.30.0-4

And there was one report of it being flaky on i386
but I wasn't able to reproduce.
See https://launchpad.net/bugs/1795556

Bug: https://bugzilla.gnome.org/785319
Applied-upstream: no
---
 clutter/tests/conform/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/clutter/tests/conform/Makefile.am b/clutter/tests/conform/Makefile.am
index 0f4681f..28fcbb5 100644
--- a/clutter/tests/conform/Makefile.am
+++ b/clutter/tests/conform/Makefile.am
@@ -26,7 +26,6 @@ actor_tests = \
 	actor-layout \
 	actor-meta \
 	actor-offscreen-limit-max-size \
-	actor-offscreen-redirect \
 	actor-paint-opacity \
 	actor-pick \
 	actor-shader-effect \
